migration-tools/migrate-rosdistro.py

```diff
@@ -114,6 +114,9 @@ def write_tracks_file(tracks, commit_msg=None):
 workdir = tempfile.mkdtemp()
 os.chdir(workdir)
 os.environ['ROSDISTRO_INDEX_URL'] = rosdistro_index_url
+os.environ['BLOOM_SKIP_ROSDEP_UPDATE'] = '1'
+
+subprocess.check_call(['rosdep', 'update'])
```

Comment:
It may be worth adding a note either in a comment or print statement to specify that this is required after the rosdistro_index_url change.

Reply:
06e9828